Tools/TestWebKitAPI/Tests/WebKitCocoa/FindInPageAPI.mm

```diff
@@ -33,6 +33,13 @@
 #import <WebKit/WKWebView.h>
 #import <wtf/RetainPtr.h>
 
+#if ENABLE(IMAGE_ANALYSIS)
+#import "ImageAnalysisTestingUtilities.h"
+#import "InstanceMethodSwizzler.h"
+#import <WebKit/_WKInternalDebugFeature.h>
+#import <pal/spi/cocoa/VisionKitCoreSPI.h>
```

Comment:
Nit - I think you can leave out this import, if you're already importing `VisionKitCoreSoftLink.h`

Reply:
Fixed!